engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -749,7 +739,7 @@ - (BOOL)stateIsBackground {
   if (flutterApplication) {
     isBackground = [self isApplicationStateMatching:UIApplicationStateBackground
                                     withApplication:flutterApplication];
-  } else if (@available(iOS 13.0, *)) {
+  } else {
```

Comment:
Same here. Relatedly, the comment on `isBackground` appears to be obsolete now, because there is no case now where we can't get the state.

Reply:
Done... but even more compacted.